### ISSUE TITLE
Vitess and PMM

### ISSUE THREAD

**Issue by schneiderdba** [vitessio/vitess#4918]
It would be really nice to have Vitess as a part of the Percona Monitoring and Management (PMM) monitoring system.  I have been working with PMM for the last 2 years and now am working on a Vitess integration at Roblox.  

#### Feature Description

Work with Percona to get Vitess supported at a "plugin" or permanent part of the PMM suite.   

#### Use Case(s)

In particular I'd like to see VTGate integrated with the Query Analytics views to make it easier for engineers to troubleshoot their application(s) without searching and or correlating MySQL instances to VTGate(s).

**Comment by schneiderdba** [comment]
![Screen Shot 2019-06-10 at 7 55 41 AM](https://user-images.githubusercontent.com/3700513/59204791-38542e00-8b56-11e9-8d4f-7878b957aad4.png)

**Comment by derekperkins** [comment]
I'm interested as well. https://jira.percona.com/browse/PMM-1963

I wrote the helm integration to launch PMM as an installation option, but still haven't added Vitess specific metrics to it.

I know that people have created some Grafana dashboards already (#3964), so it'll just take packaging them together. I've been waiting on PMM 2.0 to land before doing any further work.
